studio-api/components/IoHandler/index.js

```diff
@@ -148,15 +148,47 @@ class IoHandler extends Component {
         this.removeSocketFromRoom(roomId, socket)
       })
 
-      socket.on("watch_organization_session", (orgaId) => {
+      socket.on("watch_organization_session", async (orgaId) => {
+        // Security: Verify user has access to this organization
+        try {
+          const { isAuth, userId } = await auth_middlewares.checkSocket(socket)
+          if (!isAuth || !userId) {
+            debug(`[Session] Unauthorized watch_organization_session attempt for org ${orgaId}`)
+            return
+          }
+          const orgAccess = await model.organizations.getByIdAndUser(orgaId, userId)
```

Comment:
Fix the public session bug caused by adding a security check for organization access.  
Public users are not part of the organization and should still access the status of the session without requiring notification of the organization ID.  

Add `orgaId` in the public session token and validate it against the `organizationId` stored in the token.  
The initial public token is generated in `components/WebServer/controllers/session/session.js`, where `session.organizationId` contains the organization ID data.